lib/liftoff/scheme_builder.rb

```diff
@@ -0,0 +1,53 @@
+module Liftoff
+  class SchemeBuilder
+    def initialize(xcode_project, config)
+      @xcode_project = xcode_project
+      @config = config
+    end
+
+    def create_schemes
+      @xcode_project.generate_default_scheme
+    
+      config_schemes.each do |scheme_config|
+        name = name_from_scheme_config(scheme_config)
+        generate_scheme(name, @xcode_project, scheme_config["actions"])
```

Comment:
Have you considered not passing  `@xcode_project` as an argument since you can access it directly in the method through `@xcode_project`?